components/nodemanager-service/pgdb/node-managers-process-node.go

```diff
@@ -111,9 +115,19 @@ func (db *DB) ProcessIncomingNode(node *manager.NodeMetadata) error {
 		node.Uuid = id.String()
 	}
 
+	// trim the region field, b/c we want to just use "us-east-1", not more specific regions like "us-east-1a"
+	// this is to ensure that we don't create duplicate nodes
+	if match, err := regexp.MatchString(`^.*-.*-\d\D`, node.GetSourceRegion()); match {
```

Comment:
This seems like it could have some weird/unintended side effects, but I can't really think of a better way to do this. Anything like a hardcoded list would get outdated, etc. 🤔

Reply:
The extra letter is because it is actually the placement availability zone. The region is also in the ohai data, and we could grab this directly and it would not have the extra letter on the end.

Reply:
Here is an ohai data example with region: https://github.com/chef/automate/blob/master/components/ingest-service/chef-load-data/ohai-data.json#L980

Reply:
ya i don't love that i have to do this -- sounds like if we grab the actual region (i didn't know we had one from the chef client report) i will be able to delete this part!